<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointForm</class>
 <widget class="QWidget" name="PointForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>latitude:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_latitude">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>longitude:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_longitude">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>hemisphere:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_hemisphere">
     <property name="text">
      <string>45</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
